app/javascript/components/close_contact/CloseContact.js

```diff
@@ -288,11 +288,9 @@ class CloseContact extends React.Component {
     return (
       <React.Fragment>
         {!this.props.close_contact.id && (
-          <div className="pl-2">
-            <Button onClick={this.toggleModal}>
-              <i className="fas fa-plus"></i> Add New Close Contact
-            </Button>
-          </div>
+          <Button onClick={this.toggleModal}>
+            <i className="fas fa-plus"></i> Add New Close Contact
```

Comment:
I know this wasn't anything you changed, but I think it's appropriate to include here: I think we want to add the `fa-fw` class here. It's included for the icons in the Add New Vaccination and Add New Report buttons, and it slightly changes the alignment of the icon. 
![image](https://user-images.githubusercontent.com/8235974/110807893-76adde00-8251-11eb-92d1-c766070144e4.png)
![image](https://user-images.githubusercontent.com/8235974/110807931-86c5bd80-8251-11eb-8e81-44b2e5aa43eb.png)

If you wouldn't mind, we'd want the same change for the Add New Lab Result button:
https://github.com/SaraAlert/SaraAlert/blob/1367-monitoree-table-consistency/app/javascript/components/laboratory/Laboratory.js#L168

Reply:
done here: https://github.com/SaraAlert/SaraAlert/pull/746/commits/df2d818473729edbd96a11d0708c87d86243f429